diff-java-tool/config/checkstyle/checkstyle.xml

```diff
@@ -888,7 +888,9 @@
         <module name="ExecutableStatementCount">
             <property name="max" value="30"/>
         </module>
-        <module name="LambdaBodyLength"/>
+        <module name="LambdaBodyLength">
+            <property name="max" value="20"/>
+        </module>
```

Comment:
Not good, please move lambda content to new method

Reply:
done